### PR TITLE
Fixed invalid import data.

### PR DESCRIPTION
**Pull request by pri2si17-1997**
Signed-off-by: Priyanshu Sinha <pksinha217@gmail.com>

### CODE REVIEW

**Comment by pri2si17-1997** [comment]
Hello @teryhill @aethelwulffe 

This commit addresses #504 

I have tested it on my machine. If any empty data or invalid data is imported then it will popup an alert and nothing will happen. Else data will be saved in database.

Please check it and let me know if anything i'm missing.

File: custom/import_xml.php [hunk 1]
```diff
@@ -121,71 +121,77 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
...
-   $apatient['interpreter'],
-   $apatient['migrantseasonal'],
-   $apatient['family_size'],
-   $apatient['monthly_income'],
-   $apatient['homeless'],
-   fixDate($apatient['financial_review']),
-   $apatient['pubpid'],
-   $pid,
-   $olddata['providerID'],
-   $apatient['genericname1'],
```
Comment (aethelwulffe) [inline custom/import_xml.php]: genericname, genericval stuff should be taken out.  That is not used.  More importantly, the actual table structure should be queried (select *) for what the fields are.  Matching can be done after that.

File: custom/import_xml.php [hunk 2]
```diff
@@ -121,71 +121,77 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
...
+   		$apatient['migrantseasonal'],
+   		$apatient['family_size'],
+   		$apatient['monthly_income'],
+   		$apatient['homeless'],
+   		fixDate($apatient['financial_review']),
+   		$apatient['pubpid'],
+   		$pid,
+   		$olddata['providerID'],
+   		$apatient['genericname1'],
+   		$apatient['genericval1'],
```
Comment (aethelwulffe) [inline custom/import_xml.php]: "generic" stuff was a pre- "billing note" field usage, and no longer has a place.  Other fields like social security number have no place in usage as an importer (or anything else in the medical industry).  SSN's are only useful in the medical industry for social security claims, and that is handled in a different way.  I suggest avoiding the use of it in a standard distribution in any case (not relevant to this commit I suppose).

File: custom/import_xml.php [hunk 3]
```diff
@@ -121,71 +121,77 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
...
+
+  	setInsurance($pid, $ainsurance, $asubscriber, '1');
+  	setInsurance($pid, $ainsurance, $asubscriber, '2');
+  	setInsurance($pid, $ainsurance, $asubscriber, '3');
+  }
+  else
+  {
+  	echo "<html>\n<body>\n<script language='JavaScript'>\n";
+  	//if ($alertmsg) 
+	echo " alert('$alertmsg');\n";
```
Comment (aethelwulffe) [inline custom/import_xml.php]: Looks good.

**Pull request merged**